postgrest.cabal

```diff
@@ -114,11 +116,14 @@ library
                     , regex-tdfa                >= 1.2.2 && < 1.4
                     , retry                     >= 0.7.4 && < 0.10
                     , scientific                >= 0.3.4 && < 0.4
+                    , streaming-commons         >= 0.1.1 && < 0.3
                     , swagger2                  >= 2.4 && < 2.9
                     , text                      >= 1.2.2 && < 1.3
                     , time                      >= 1.6 && < 1.12
                     , timeit                    >= 2.0 && < 2.1
                     , unordered-containers      >= 0.2.8 && < 0.3
+                    -- pinning unix-compat to match one's version in warp
+                    , unix-compat               == 0.5.4
```

Comment:
This dependency pinning seems like it could give trouble later. Can we avoid the dep or the pinning? Why is it needed?

Reply:
Good point, unpinning it altogether. Compilation seems to succeed and tests seem to pass